docs/runbooks/LowReadyVirtControllersCount.md

```diff
@@ -0,0 +1,43 @@
+# LowReadyVirtControllersCount
+
+## Meaning
+
+From a high-level perspective the virt-controller has all the cluster wide virtualization functionality.
+
+This controller is responsible for monitoring the VMI (CRDs) and managing the associated pods. The controller will make sure to create and manage the life-cycle of the pods associated to the VMI objects.
+
+A VMI object will always be associated to a pod during it's life-time, however, due to i.e. migration of a VMI the pod instance might change over time.
+
+This alert is triggered when some virt controllers are running but not ready in the past 5 minutes.
+
+## Impact
+In these circumstances the virt-controller will become a single point of failure.
+
+## Diagnosis
+
+- Set the environment variable NAMESPACE
+
+  ```
+  export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+  ```
+
+- Run
+  ```
+  kubectl get deployment -n $NAMESPACE virt-controller -o jsonpath='{.status.readyReplicas}'
+  ```
+
+
+
+## Mitigation
```

Comment:
Sorry for checking this PR so late but I think the things here are not mitigation steps. They are mostly `diagnosis` steps. In mitigation section we should give some information about how to fix the issue. If there is no specific to mention, we can say "fix the issue found in mitigation section.otherwise, create a ticket."